vermouth/processors/do_mapping.py

```diff
@@ -45,7 +45,7 @@ class GraphMapping:
     #       of blocks and a mapping of {node_idx: [node_idx, ...], ...}
 
     # TODO: renumber output residues. Needs information about the entire system
-    #       or we need to at least garantue we run it all in order. Which we
+    #       or we need to at least the garanty we run it all in order. Which we
```

Comment:
Still typo'd ;)

guarantee

Reply:
I'll blame it on the fact that it is 1AM here.